packages/dashboard/src/styles/variables.css

```diff
@@ -8,7 +8,7 @@
   --selection-border-touch-width: 8px;
   --selection-radius-small: 4px;
   --stack-order-grid-image: -999999999;
-  --stack-order-selection-box: 9999999999999;
+  --stack-order-selection-box: 99999999;
```

Comment:
Could we make these into something more comprehensible? Hard to know what these indices are trying to accomplish with these values, and I don't think they really need to be this high.

Reply:
I've taken these from the original implementation. I'll have a look at refactoring the CSS